cdl-section3-ai-innovation.md

```
@@ -0,0 +1,1798 @@
+---
+
+## 📋 目次
+
+1. [Section 3 の出題範囲と学習ポイント](#1-section-3-の出題範囲と学習ポイント)
+2. [AI・ML の基礎概念を理解する](#2-aiml-の基礎概念を理解する)
+3. [機械学習の 3 つのアプローチ](#3-機械学習の-3-つのアプローチ)
+4. [Google Cloud AI サービスの階層構造](#4-google-cloud-ai-サービスの階層構造)
+5. [プリビルト AI API（事前学習済みモデル）](#5-プリビルト-ai-api事前学習済みモデル)
+6. [AutoML（ノーコード ML）](#6-automlノーコード-ml)
+7. [Vertex AI（統合 ML プラットフォーム）](#7-vertex-ai統合-ml-プラットフォーム)
+8. [生成 AI（Generative AI）の世界](#8-生成-aigenerative-ai-の世界)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)